angularSTEP/src/app/upload-recipe/upload-recipe.component.ts

```diff
@@ -21,7 +21,7 @@ export class UploadRecipeComponent {
 
   recipeForm = new FormGroup({
     name: new FormControl('', [Validators.required]),
-    description: new FormControl('', [Validators.required]),
+    description: new FormControl('', [Validators.required, Validators.maxLength(200)]),
```

Comment:
AFAIK this validator still needs to eventually be paired with something that checks it server-side, right?